### ISSUE TITLE
[BUG] koordlet continues reporting errors: error write cpuset.cpus: no space left on device

### ISSUE THREAD

**Issue by wangxiaoq** [koordinator-sh/koordinator#1460]
**What happened**:

koordlet continues reporting errors like:

```
I0710 16:18:11.021831  492950 container_context.go:183] set container xxx/xxx/xxx cpuset  on cgroup parent kubepods/besteffort/podb481a53f-9b99-485e-9438-f975c9dc26e7/3eb53be8450b7a1b30070b1d74fdc1892d470535c0e446fed2dfa6faecf383f5 failed, error write /host-cgroup/cpuset/kubepods/besteffort/podb481a53f-9b99-485e-9438-f975c9dc26e7/3eb53be8450b7a1b30070b1d74fdc1892d470535c0e446fed2dfa6faecf383f5/cpuset.cpus: no space left on device
```

**What you expected to happen**:

koordlet should not report these errors.

**How to reproduce it (as minimally and precisely as possible)**:

use command like below, you could find some error logs:

```
kubectl logs <koordlet-pod> -nkoordinator-system | grep "no space left"
```

**Anything else we need to know?**:

**Environment**:
- App version: 1.1.1
- Kubernetes version (use `kubectl version`): 1.18.6
- Install details (e.g. helm install args):
- Node environment (for koordlet/runtime-proxy issue):
  - Containerd/Docker version:
  - OS version:
  - Kernal version:
  - Cgroup driver: cgroupfs/systemd
- Others:

**Comment by wangxiaoq** [comment]
I find the reason is that koordlet assign BE containers with cpuset value = string("")，https://github.com/koordinator-sh/koordinator/blob/06b2f5ba2582aaab8d922f51a84d22306a642473/pkg/koordlet/runtimehooks/hooks/cpuset/rule.go#L98

The command like below could also trigger this error:

```
echo "" > /host-cgroup/cpuset/kubepods/besteffort/podb481a53f-9b99-485e-9438-f975c9dc26e7/3eb53be8450b7a1b30070b1d74fdc1892d470535c0e446fed2dfa6faecf383f5/cpuset.cpus
``` 

The output of this command is:

```
-bash: echo: write error: No space left on device
```

**Comment by wangxiaoq** [comment]
Sorry, I see it has been fixed here: https://github.com/koordinator-sh/koordinator/blob/06b2f5ba2582aaab8d922f51a84d22306a642473/pkg/koordlet/runtimehooks/protocol/container_context.go#L212C79-L212C79   , close it

**Comment by DSP-doublehappy111** [comment]
it just fixed in main branch,not in v1.2.0
